concent_api/core/tasks.py

```diff
@@ -40,6 +42,9 @@ def upload_finished(subtask_id: str) -> None:
         logging.error(f'Task `upload_finished` tried to get Subtask object with ID {subtask_id} but it does not exist.')
         return
 
+    if subtask.state_enum in Subtask.ACTIVE_STATES and subtask.next_deadline <= timezone.now():
```

Comment:
it should not be there - the handling is being done few lines below

Reply:
removed